src/kernels/MIOpenBatchNormBwdSpatial.cl

```diff
@@ -229,9 +229,23 @@ MIOpenBatchNormBwdSpatial(const __global _FLOAT* __restrict x_in,
 
 #elif(MIO_BN_VARIANT == 1)
 
+#ifndef MIO_LAYOUT_NHWC
+#define MIO_LAYOUT_NHWC 0
+#endif
+
+#if (MIO_LAYOUT_NHWC != 0) || (MIO_LAYOUT_NHWC != 1)
+#error MIO_LAYOUT_NHWC must be 0 or 1
```

Comment:
[Recommendation] Oops, I am afraid we need double quotes around error message.
```suggestion
#error "MIO_LAYOUT_NHWC must be 0 or 1"
```

Reply:
@atamazov I'm afraid I might have messed up something and trying to correct them:
```
[2021-12-16T21:18:03.373Z] 13/93 Test  #3: test_bn_3d_spatial_test ...............................***Failed  Error regular expression found in output. Regex=[FAILED]  1.34 sec
[2021-12-16T21:18:03.373Z] Choosing smaller input values for low dims
[2021-12-16T21:18:03.373Z] MIOpen(OpenCL): Error [BuildProgram] Build log: /tmp/comgr-547839/input/CompileSource:446:2: error: "MIO_LAYOUT_NHWC must be 0 or 1"
[2021-12-16T21:18:03.373Z] #error "MIO_LAYOUT_NHWC must be 0 or 1"
[2021-12-16T21:18:03.373Z]  ^
[2021-12-16T21:18:03.373Z] 1 error generated.
[2021-12-16T21:18:03.373Z] Error: Failed to compile opencl source (from CL or HIP source to LLVM IR).
```

Reply:
Maybe #error macro is not accepted in COMGR or hipRTC?

Reply:
OMG we shall use `&&`, not `||` 🤕

Reply:
@atamazov ah, I'm so distracted and should have figured it out sooner :) I'm going to change it really soon.